mache-couchbase/src/main/java/com/excelian/mache/couchbase/builder/CouchbaseProvisioner.java

```diff
@@ -99,10 +110,26 @@ public CouchbaseProvisionerBuilder withSchemaOptions(SchemaOptions schemaOptions
             return this;
         }
 
-        public CouchbaseProvisioner create() {
-            return new CouchbaseProvisioner(couchbaseEnvironment, bucketSettings, nodes, adminUser, adminPassword,
-                    schemaOptions);
+        @Override
+        public CouchbaseProvisionerBuilder withDefaultSchemaOptions() {
+            schemaOptions = SchemaOptions.USE_EXISTING_SCHEMA;
+            return this;
+        }
+
+        @Override
+        public AdminSettings withBucketSettings(BucketSettings bucketSettings) {
+            this.bucketSettings=bucketSettings;
+            return this;
         }
+
+        public CouchbaseProvisioner build() {
+            if(cluster==null)
```

Comment:
Actually, ensuring a cluster should probably be enforced by the builder - is it? - if so, bin this check